experiment/example/example.go

```diff
@@ -0,0 +1,68 @@
+// Package example contains a simple example of experiment.
+package example
+
+import (
+	"context"
+	"time"
+
+	"github.com/ooni/probe-engine/experiment"
+	"github.com/ooni/probe-engine/experiment/handler"
+	"github.com/ooni/probe-engine/model"
+	"github.com/ooni/probe-engine/session"
+)
+
+const (
+	testName    = "example"
+	testVersion = "0.0.1"
+)
+
+// Config contains the experiment config.
+type Config struct {
+	sleepTime time.Duration
+}
+
+// TestKeys contains the experiment's result.
+//
+// This is what will end up into the Measurement.TestKeys field
+// when you run this experiment.
+type TestKeys struct {
+	foo     string `json:"foo"`
+	success bool   `json:"success"`
+}
+
+func measure(
+	ctx context.Context, sess *session.Session, measurement *model.Measurement,
+	callbacks handler.Callbacks, config Config,
+) error {
+	ctx, cancel := context.WithTimeout(ctx, 30*time.Second)
+	defer cancel()
+	config.sleepTime = 10 * time.Second
+	testkeys := &TestKeys{
+		foo:     "bar",
+		success: false,
+	}
+	measurement.TestKeys = testkeys
+	dosomething(config)
```

Comment:
I think it's fine to sleep directly here without deferring to a secondary function